components/crud-web-apps/volumes/Dockerfile

```
@@ -22,7 +22,7 @@ COPY ./common/frontend/kubeflow-common-lib/tsconfig.json .
 RUN npm run build
 
 # --- Build the frontend ---
-FROM node:16.20.2-bullseye as frontend
+FROM node:24-bookworm as frontend
```

for more _deterministic builds_ - I think we should fix this new version to a major/minor/patch version just like it was originally.

```suggestion
FROM node:24.11.1-bookworm as frontend-kubeflow-lib
```